### DISCUSSION TITLE
`brew --prefix` isn't documented (anywhere I could find)

### DISCUSSION THREAD

**Discussion by OLDlayeratwork** [Homebrew/discussions#680] [Everyday usage] [answered]
`brew --help` doesn't show it.  I looked all over brew.sh.  It should be really nice if `brew --help` mentioned it because I was looking for a way to query the prefix, given the new M1 installation we have, which is in `/opt/homebrew` and I want to fix up some scripts.

Thank you.

**Comment by carlocab** [comment, marked as answer]
Have you tried [`man brew`](https://docs.brew.sh/Manpage#--prefix---unbrewed---installed-formula-)?

**Reply by Rylan12** [reply]
Also, [the online documentation](https://docs.brew.sh/Manpage#--prefix---unbrewed---installed-formula-).

Not sure what else you're looking for here. `brew --help` outputs the following lines:

```
Further help:
  brew commands
  brew help [COMMAND]
  man brew
  https://docs.brew.sh
```

All of these options will show you the `--prefix` command.

**Reply by OLDlayeratwork** [reply]
Thanks, both of you.

**Comment by OLDlayeratwork** [comment]
OK, sorry for the noise.  I had `MANPATH` set and `man brew` couldn't find a man page, and I missed `brew commands`.

Thanks.
